pkg/acsengine/engine.go

```diff
@@ -146,7 +146,7 @@ func FormatAzureProdFQDN(fqdnPrefix string, location string) string {
 	return fmt.Sprintf("%s.%s."+FQDNFormat, fqdnPrefix, location)
 }
 
-//getCloudSpecConfig returns the kubenernetes container images url configurations based on the deploy target environment
+//getCloudSpecConfig returns the kubernetes container images url configurations based on the deploy target environment
```

Comment:
if you want to go one step further you could capitalize `Kubernetes` and `URL`

Reply:
Done! lol

Reply:
🍾